eng/pipelines/template-tests.yml

```diff
@@ -45,38 +117,16 @@ jobs:
       name: azsdk-pool-mms-ubuntu-2004-general
       vmImage: MMSUbuntu20.04
 
+    container: $[ variables['AzdContainerImage'] ]
+
     steps:
       - checkout: self
 
-      - template: /eng/pipelines/templates/steps/install-bicep.yml
```

Comment:
Why are we removing these CLI installations?

Reply:
I was trying to move us towards using our published dev containers. I think that completes the end-to-end verification loop for adding a template in `azd`. Change has been reverted and will come in as a separate PR.